votor/src/certificate_pool.rs

```diff
@@ -686,6 +689,7 @@ impl CertificatePool {
     /// This avoids the need to recreate the entire certificate pool since it's
     /// not distinguished by the pubkey.
     pub fn update_pubkey(&mut self, new_pubkey: Pubkey) {
+        self.my_pubkey = new_pubkey;
```

Comment:
Not relevant for this PR: err, this is our set-id implementation currently?

Guess we need to update safe_to_notar and safe_to_skip when pubkey changes to filter self-votes from the pool?

Reply:
yep, currently we don't handle set identity at all for the certificate pool, but we should plug it in at some point #289

Reply:
When set-id is done, would vote history be updated to pull all the votes done by the same pubkey but by another instance from the cert pool?